### DISCUSSION TITLE
Request New features include

### DISCUSSION THREAD

**Discussion by ecomdeveloperbd** [inspireui/support#5701] [New Feature/Ideas Requests]
Fluxstore all app... 
I have been requesting to add this feature for a long time. Hope to add the features very soon
 
 
1. follow this video 
 
https://streamable.com/k0m1tc
 
add this feature. Vendor shop list follow this layout section... All shop list gridview styles this... (👇 follow this symbol section )
 
2. Email verification
 
 
3. Add Product badges plugin 
 
4. please add Bangladesh most popular payment gateway Bkash or sslcommerz, nagad
 
5. From WordPress site to app product loading speed heavy slow... ( i will requested how to faster product loading in apps optimize code and api ). Like : Fluxstore Shopify in app product loading super fast. but WordPress very slow. please improve Fluxstore wocommerce apps.
 
6. Before each product and category or any other item is loaded, the website or custom logo should be preloaded. ( static logo or animation or gif any logo is set ) This feature can also be found in 8 out of 10 apps currently. Please add this feature
 
 
7. If there is no internet connection, the pop-up will still give a warning. Your device requires an internet connection Please connect to the internet (animation like this)

8.Product layout - Auto Sliding. it's possible. Timing Set like banner style - auto slide
product layout - how to set auto slide style

https://streamable.com/ds3fvn

9. login page video background set ....
https://streamable.com/nogqs9

10. Gridview or listview configuration store design UI with home page show multiple row store names and view more store option add. and all shop names show another activity page. ( Gridview and set two or more columns )

Follow demo UI some screenshots
https://i.imgur.com/jIQpg2l.jpg

11. when multiple dynamic 5-10+ page include single Tabmenu ..... every tab menu option click then change dynamic page....
 
 
i would suggest please. it's Tabmenu option add swipe option.... just swipe change the dynamic page from Tabmenu.
 
example: swipe option... show the video
Link: https://streamable.com/8glky6

12.  Every product details under related product row increase..... like 2, 3 or more show more products and under new layout add recently view this products show and top selling products show... and upsell crose sell options adds

13. Integration any email marketing tools like : MailChimp. Identify customer visit and search product. ( pop offer send ) in apps or notification. and tracking customer behavior.
 
 This  features for Fluxstore Shopify

This feature is very important for Shopify apps
1. Currency conversion option – very important for dropshipping
2. Photo review option is more important – very important for dropshipping
3. Any marketing tools integration (SMS Marketing & Email) – -very important for dropshipping
4. from visitor traffic source from apps - analysis
5. AI Camera or Image search – Product search
 
I am also asking for your benefit and for adding new features to the product. I want your products to be on the top list.

**Comment by osman-kocak** [comment]
+Search page gride layout
